frontend/src/styles/generic/layout.scss

```diff
@@ -52,19 +52,18 @@ body {
     margin-top: 5px;
     height: 50%;
   }
-  
-  div {
-    display: fixed;
-    padding: $default-padding-sm;
+
+  & div {
+    margin-right: 5px !important;
```

Comment:
I'll repeat the question just to be sure that the `!important` is definitely needed?